macaroonbakery/json_serializer.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+import json
+import utils
+
+from pymacaroons.utils import convert_to_bytes
+from pymacaroons.macaroon import Macaroon
+from pymacaroons.caveat import Caveat
+
+
+class JsonSerializer(object):
+    def serialize(self, macaroon):
+        serialized = {
+            'identifier': macaroon.identifier,
+            'signature': macaroon.signature
+        }
+        if macaroon.location:
+            serialized['location'] = macaroon.location
+        if macaroon.caveats:
+            serialized['caveats'] = [
+                to_dict(caveat) for caveat in macaroon.caveats]
```

Comment:
For formatting conistency with round brackets, maybe the closing ] should be on a separate line here.